### DISCUSSION TITLE
Migrarazione Ha Pi4

### DISCUSSION THREAD

**Discussion by Al3eRtO** [indomus/forum#2029] [HUB: Home Assistant]
Buonasera a tutti, 
sarà una domanda banale ma la faccio ugualmente essendo ancora alle prime armi.
Attualmente ho homeAssistant Os su raspberry Pi4 1gb installato su ssd. Ho recuperato un nuovo Pi 4 con 4gb di ram e volevo sapere la procedura e le cose da fare per il trasferimento. Ho configurato il nuovo rasp per leggere l'ssd, basta solo collegare e avviare o devo fare altro?
Grazie

**Comment by g4mbr1nu5** [comment]
Ciao,
Io ho appena eseguito una migrazione da Raspberry 3 a Virtual Machine dedicata e ho usato l'add-on per eseguire il backup su google drive:
https://github.com/sabeechen/hassio-google-drive-backup

Fai il backup del vecchio raspberry, installi una HA vuota sul nuovo, al primo avvio, quando ti chiede di creare l'utente, in basso trovi il tasto per caricare il backup.
Una volta caricato riavvii e ti ritrovi tutto ripristinato...almeno per me ha funzionato.

Nel link comunque c'è scritto tutto

**Comment by Al3eRtO** [comment]
Fatto, l'unica cosa che non va è il collegamento da remoto, usavo duckdns con portF e basta. Sembra tutto ok ma non va
